### ISSUE TITLE
SharePoint Online Webhook with an intranet notification endpoint

### ISSUE THREAD

**Issue by SouravBiki** [SharePoint/sp-dev-docs#6231]
## Category

- [x] Question
- [ ] Typo
- [ ] Additional article idea
- [ ] Checked items have no spaces between the 
I am having a requirement where I have to use a intranet web api url as notification url for SharePoint Online webhook. The web api is hosted on an IIS server and the authentication is set to anonymous. Is it at all possible without the use of ngrok?

## Question

> **DELETE THIS LINE BEFORE SUBMITTING** | *If you are reporting a bug, please use our [bug issue template](new?assignees=&labels=Needs%3A+Triage&template=bug_report.md&title=). If you are suggesting an enhancement please describe thoroughly the enhancement, how it can be achieved, and the expected benefit.*

## Environment details (*development & target environment*)

> **DELETE THIS LINE BEFORE SUBMITTING** | *Complete the following (if ignored you'll be prompted for it before we can address your issue... save the time and provide it in your initial submission).*

- **Your Developer Environment**: [e.g. Windows 10 | MacOS 10.15.x]
- **Target Environment**: [e.g. SharePoint Server 2016 | SharePoint Server 2019 | SharePoint Online]
- **Framework**: [e.g. .NET Framework v3.x | .NET Core v3.x | Node.js v6/v8/v10/v12]
- **Browser(s)**: [e.g. Chrome v79 | Safari v22 | Firefox v31 | IE v11 | Edge | Chromium Edge v79]
- **Tooling**: [e.g. VS Code | SPFx v1.10.0 | Visual Studio 2019]
- **Additional details**: *The more context you can provide, the easier it is (and therefore quicker) to help.*

## Submission guidelines

> - **!!IMPORTANT!!** - All submissions must complete the baseline sections included in this template. Ignoring or deleting this template may result in closing the issue with the label **type:incomplete-submission**.
> - Follow our guidance on [How To Create Good Issues](https://github.com/sharepoint/playground-repo/wiki/How-to-Create-Good-Issues).
> - If you are reporting an issue around any of the documents or articles, please ensure that you have referenced the specific file or URL of the related document.
> - If you have technical questions about the framework, we monitor the tags #spfx, #spfx-webparts, and #spfx-tooling on [SharePoint StackExchange](http://sharepoint.stackexchange.com).
> - Remember to include sufficient details and context.
> - If you have multiple questions, suggestions, or bugs, please submit them in separate issues.
>
> **DELETE THIS SECTION BEFORE SUBMITTING**
>
> *Thanks for your contribution! Sharing is caring.*

**Comment by andrewconnell** [comment]
Ngrok is just used when developing webhooks... it simplifies the development process because when registering a webhook endpoint, you must provide a routable HTTPS endpoint. Usually, you're developing on your local dev environment (http://localhost) which isn't routable/reachable from SPO. So, ngrok creates a temporary routable URL that redirects to your local dev machine.

To answer your question:

> The web api is hosted on an IIS server and the authentication is set to anonymous.

Yes, you can do this provided the URL is routable & secured. 

BTW, in the future please complete the requested information in the new issue template. You ignored the environment section which provides additional details about your environment that is helpful in troubleshooting or understanding context. In the future, issues that ignore this section are closed as incomplete submissions.
